uplink/ecclient/client.go

```diff
@@ -148,6 +149,18 @@ func (ec *ecClient) Put(ctx context.Context, limits []*pb.AddressedOrderLimit, p
 	}()
 
 	successes := int(atomic.LoadInt32(&successfulCount))
+	mon.IntVal("segment_pieces").Observe(int64(len(limits)))
```

Comment:
could this be pieceCount instead of len(limits)?

Reply:
It should be, thanks!